src/datanode/src/datanode.rs

```diff
@@ -39,6 +39,13 @@ pub enum ObjectStoreConfig {
         endpoint: Option<String>,
         region: Option<String>,
     },
+    OSS {
+        bucket: String,
+        root: String,
+        access_key_id: String,
+        access_key_secret: String,
```

Comment:
Do we need  to keep this field the same with S3's config above?

```suggestion
        secret_access_key: String,
```

Reply:
Hi @waynexia , in the opendal's implementation, the name for this field is not consistent with S3. I am happy to change it to `secret_access_key` if you want.

Reply:
I did some search and it looks like this difference is coming from OSS and S3 official SDKs. Let's leave it unchanged...